addon/ActivityStreams.js

```diff
@@ -415,6 +418,21 @@ ActivityStreams.prototype = {
     }
   },
 
+  _setupTopSitesTwoRowsExperiment() {
+    this._experimentProvider.on("experimentEnrolled", this._handleExperimentEnrollment);
+  },
+
+  _unsetupTopSitesTwoRowsExperiment() {
+    this._experimentProvider.off("experimentEnrolled", this._handleExperimentEnrollment);
+  },
+
+  _handleExperimentEnrollment(eventName, experiment) {
+    // If we are enrolling into the topSitesTwoRows experiment, turn on the showMoreTopSites pref.
+    if (experiment.id === "topSitesTwoRowsDefault") {
+      simplePrefs.prefs.showMoreTopSites = true;
```

Comment:
Just move this right to here:

https://github.com/mozilla/activity-stream/blob/master/addon/ExperimentProvider.js#L73

instead of having it in ActivityStream.js

Christian ran into annoying timing issues and we don't need to be fancy with it, it's fine if it just lives right in the experiment provider.